src/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { tap, take, runEffects } from "@most/core";
+import { newDefaultScheduler } from "@most/scheduler";
+import { describe, it } from "mocha";
+import assert from "power-assert";
```

Comment:
Let's stick with [`@briancavalier/assert`](https://www.npmjs.com/package/@briancavalier/assert), since the test case is so nice and simple (👏 )